.github/workflows/test-schema.yml

```diff
@@ -0,0 +1,49 @@
+name: Test model schemata
+
+on:
+  push:
+    paths:
+      - dandi/model*.py
+      - tools/pubschemata.py
+  pull_request:
+    paths:
+      - dandi/model*.py
+      - tools/pubschemata.py
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Need history for `git describe`
+          path: dandi
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
+
+      - name: Install dandi
+        run: python -m pip install .
+        working-directory: dandi
+
+      - name: Checkout dandi/schema
+        uses: actions/checkout@v2
+        with:
+          repository: dandi/schema
+          path: schema
+
+      - name: Generate model schemata
+        run: python dandi/tools/pubschemata.py schema/releases
+
+      - name: Commit changes
+        run: |
+          if git status --porcelain | grep -q "^.M"
```

Comment:
what if new files get  added, removed or renamed - I believe this would not catch it. I think it should boil down to ensuring no changes whatsoever to any of the previously existed folders, so getting a list of folders and running `git status --porcelain` on all of them and then grep'ing for any indicator.

Reply:
Updated.